interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
Mmmm, totally - I agree. In fact, there are a couple other places in the code where I could either use this other approach, or wrap statements in the for loop inside an ``if`` statement with a ``hasOwnProperty()`` check, so I will go ahead and do this.